docfx/docs/Examples.md

```diff
@@ -0,0 +1,188 @@
+# Code examples
+
+When, for example, marshaling is enabled and `useSafeHandles` is `true`, the code can be different than that in C++. Here we show a few code examples of using CsWin32.
```

Comment:
I just realized, in reading this, that there's no SafeHandle examples. Could you include some? If not, I can do it as a follow-up.

Reply:
The first sample uses SafeHandle

https://github.com/microsoft/CsWin32/blob/e58c9574e7bed0236242e24e468df591d1a8f744/docfx/docs/Examples.md?plain=1#L30-L45